### PR TITLE
supports_all_drives was added to Client.create - whether to look for …

### PR DESCRIPTION
**Pull request by yohayonyon**
…the folder_id in shared drives.

### CODE REVIEW

**Comment by burnash** [comment]
Thank you for the PR. Could you move the last commit (2d72f12) into a separate PR? It'll be easier to test and also it'll be better documented that way.

**Comment by yohayonyon** [comment]
Not so elegant, but done. :-)

File: gspread/client.py [hunk 1]
```diff
@@ -193,7 +194,13 @@ def create(self, title, folder_id=None):
         if folder_id is not None:
             payload['parents'] = [folder_id]
 
-        r = self.request('post', DRIVE_FILES_API_V3_URL, json=payload)
+        params = None
+        if supports_all_drives:
+            params = {
+                'supportsAllDrives': True
+            }
```
Comment (lavigne958) [inline gspread/client.py:201]: It works if we set the parameter to `True` even if the user want to save the file in its own drive.

I recommend here, to always create variable `params` and set the value of `'supportsAllDrives'` to `supports_all_drives`

**Comment by lavigne958** [comment]
Thank you @yohayonyon for this contribution, could you please squash all your commits to only include the part that is fixed by your PR please ?
If you need any help doing this operation (as it can be dangerous) let me know I'll guide you.

**Comment by lavigne958** [comment]
closes #778

**Comment by lavigne958** [comment]
Re-created the pr in a simpler way, we can enable the sharedDrive option all the time it has no impact on regular drives and allows to create a spreadhseet in a shared drive.

Closing this PR in favor of #895

**Pull request closed without merging**